uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/BrokerSupportRule.java

```diff
@@ -61,7 +62,7 @@ public Statement apply(final Statement base, final Description description) {
         return new Statement() {
             @Override
             public void evaluate() throws Throwable {
-                Log.i(TAG, "Applying rule....");
+                Logger.i(TAG, "Applying rule....");
```

Comment:
Here's an opportunity: instead of using static references inline, can we take a per-class instance as a static variable?  That would align us with normal logging anyway.

Reply:
As discussed on chat the other day....keeping static references for now....we could do this later in a separate PR if needed.